docs/how-to-work-with-conf-files.md

```diff
@@ -31,28 +27,46 @@ WorldDatabaseInfo     = "127.0.0.1;3306;root;root;azerothcore_test_world"
 CharacterDatabaseInfo = "127.0.0.1;3306;root;root;azerothcore_test_chars"
 ```
 
-### Modules case
+## Modules configuration
+
+After normal `.conf` and `.conf.dist` files has been loaded, you are able to load an infinite number of configuration files using scripts/modules API. They will have the same behaviour as described above. 
 
-After normal .conf and .conf.dist files has been loaded, you're able to load infinite other configuration files using scripts/modules API. They will have the same behaviour of .conf described above. However, we do not recommend you to overwrite server configuration properties since you can have concurrency issues with other modules that use them too, **create new namespaced properties instead**.
+**Note:** we do not recommend you to overwrite server configuration properties since you can have concurrency issues with other modules that use them too. Instead, **create new namespaced properties**.
```

Comment:
I would consider using a capital W since it is the beginning of a sentence.
```suggestion
**Note:** We do not recommend you to overwrite server configuration properties since you can have concurrency issues with other modules that use them too. Instead, **create new namespaced properties**.
```